pyccel/codegen/printing/fcode.py

```diff
@@ -599,13 +599,16 @@ def _print_Lambda(self, expr):
 #
 #        return self._get_statement(code)
 
+    def print_kind(self, expr):
+        return ""+iso_c_binding[self._print(expr.dtype)][expr.precision]
```

Comment:
Why do we need the `""+`?

Reply:
we dont need it, since we get a string from ISO_C_BINDING.